src/test/java/picard/analysis/TheoreticalSensitivityTest.java

```diff
@@ -128,7 +128,7 @@ public void testCentralLimitTheorem() throws Exception {
         //test whether the number of elements within one standard deviation agrees with the normal distribution
         final List<Double> thresholds = Arrays.asList(mu - sigma, mu + sigma);
 
-        //sums is 1 x sampleSize, thresholds is a 2-vector, so proportions is 1 x 2
+        //sums is 1 x theoreticalHetSensitivitySampleSize, thresholds is a 2-vector, so proportions is 1 x 2
```

Comment:
I don't think this should have changed here. since you still have sampleSize in the code above.